src/types.ts

```diff
@@ -1,5 +1,45 @@
-import { FunctionInput } from "@azure/functions";
+import { FunctionInput, FunctionHandler, FunctionOutput, FunctionTrigger } from "@azure/functions";
+import { IEntityFunctionContext } from "../src/ientityfunctioncontext";
+import { IOrchestrationFunctionContext } from "../src/iorchestrationfunctioncontext";
 
+// orchestrations
+export type OrchestrationHandler = (
+    context: IOrchestrationFunctionContext
+) => Generator<unknown, unknown, any>;
+
+export type OrchestrationOptions = {
```

Comment:
This might work for activities, but for entities and orchestrations, won't that allow them to add extra inputs and extra outputs? Which they shouldn't be allowed to do

Reply:
Done